js/util/evented.js

```diff
@@ -78,7 +78,14 @@ var Evented = {
      * @returns {Object} `this`
      */
     fire: function(type, data) {
-        if (!this.listens(type)) return this;
+        if (!this.listens(type)) {
+            // Error events are important for debugging and should not be
+            // silently dropped.
+            if (util.endsWith(type, 'error')) {
```

Comment:
Yeah, makes sense. I'm not very concerned, just thought it would be slightly less hacky/simpler. Maybe just `if (data && data.error)` would do?